scripts/workers/submission_worker.py

```diff
@@ -446,20 +446,23 @@ def main():
     sys.path.append(COMPUTE_DIRECTORY_PATH)
 
     load_active_challenges()
-
     connection = pika.BlockingConnection(pika.ConnectionParameters(
-        host='localhost'))
+        host=RABBITMQ_PARAMETERS.get('HOST', None)))
 
     channel = connection.channel()
-    channel.exchange_declare(exchange='evalai_submissions', type='topic')
+    channel.exchange_declare(
+        exchange=RABBITMQ_PARAMETERS.get('EXCHANGE_NAME', None),
+        type=RABBITMQ_PARAMETERS.get('EXCHANGE_TYPE', None))
 
     # name can be a combination of hostname + process id
     # host name : to easily identify that the worker is running on which instance
     # process id : to add uniqueness in case more than one worker is running on the same instance
     add_challenge_queue_name = '{hostname}_{process_id}'.format(hostname=socket.gethostname(),
                                                                 process_id=str(os.getpid()))
 
-    channel.queue_declare(queue='submission_task_queue', durable=True)
+    channel.queue_declare(
+        queue=RABBITMQ_PARAMETERS.get('SUBMISSION_QUEUE', None).get('NAME', None),
+        durable=RABBITMQ_PARAMETERS.get('SUBMISSION_QUEUE', None).get('DURABLE', None))
```

Comment:
No need of taking durable key from settings, this is something which will vary from queue to queue, so this much of arguments can be kept in worker only.